spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/StreamPropertyKeys.java

```diff
@@ -23,7 +23,7 @@
  */
 public class StreamPropertyKeys {
 
-    // @formatter:off (not working)
+	// @formatter:off (not working)
     static final String PREFIX = "spring.cloud.stream.";
```

Comment:
do we know why the formatting isn't working on this class? seems like a simple class that should be easy to format